Rebuild-Black-Business/RBB-Website
# remove duplicate close button from modals

# Describe your PR

Related to # https://trello.com/c/PLqBskue/208-remove-large-close-button-on-submit-business-form
Fixes #

It looks like that's where the "submit" button would typically go, users might click it erroneously

Users can already hit 'x' or tap/click outside the modal to dismis

## Pages/Interfaces that will change
Business Form Modal and other models that had both an 'x' and close button

### Screenshots / video of changes


Drop some screenshots of the before and after of your work here. Better yet, take a screen recording using a tool like [Loom](https://www.loom.com/)

Before
![image](https://user-images.githubusercontent.com/6998954/84442333-6d80d980-ac0b-11ea-9023-91163e13a4d1.png)


After
<img width="1241" alt="Screen Shot 2020-06-11 at 5 44 21 PM" src="https://user-images.githubusercontent.com/6998954/84442328-6954bc00-ac0b-11ea-97b2-b9b95ea71be6.png">


## Steps to test

1. Go to /businesses
2. Click "Register a business" button
3. Click 'X' or outside of modal to close modal and no longer see  the close button

### Additional notes
